libraries/AP_Motors/AP_MotorsUGV.h

```diff
@@ -114,6 +117,11 @@ class AP_MotorsUGV {
     //  returns true if checks pass, false if they fail.  display_failure argument should be true to send text messages to GCS
     bool pre_arm_check(bool report) const;
 
+    // return the motor mask
+    uint16_t get_motor_mask() const { return _motor_mask; }
+
+    uint8_t get_pwm_type() const { return _pwm_type; }
```

Comment:
can we add a comment above "get_pwm_type()"?

Reply:
we should also consider having it return the AP_MotorsUGV::pwm_type although that's optional.

Reply:
Fixed the comment. That's what it returns already?